program/tests/tests/withdrawals.rs

```diff
@@ -4,6 +4,7 @@
 #![cfg(feature = "test-bpf")]
 
 use bincode::deserialize;
+use solana_program::program_error::ProgramError;
```

Comment:
Is this change needed?

Reply:
Good catch, I was testing some things and that was left.